examples/scheduled-transaction-multi-sig-threshold.js

```diff
@@ -0,0 +1,109 @@
+const {
+  Client,
```

Comment:
Can you update this file to use 4 spaces at tab. We have an `.editorconfig` file at the root of the project, it might be easier for you to add some plugin to your editor that automatically reads this file, and reformat that way.